api/v1/controllers/EventController.js

```diff
@@ -0,0 +1,154 @@
+var bodyParser = require('body-parser');
+var _Promise = require('bluebird');
+
+var errors = require('../errors');
+var services = require('../services');
+var config = require('../../config');
+
+var middleware = require('../middleware');
+var requests = require('../requests');
+var utils = require('../utils');
+
+var EventService = require('../services/EventService');
+
+var logger = require('../../logging');
+
+var router = require('express').Router();
+
+function getEvents(req, res, next) {
+        var updated = req.query.lastUpdated || 0;
```

Comment:
I don't think any of our endpoints return epoch milliseconds. They all return unix timestamps